qpp-data-restoration/src/main/java/gov/cms/qpp/conversion/qppdatarestoration/QppDataRestorationApplication.java

```diff
@@ -0,0 +1,79 @@
+package gov.cms.qpp.conversion.qppdatarestoration;
+
+import com.amazonaws.regions.Regions;
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB;
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBClientBuilder;
+import com.amazonaws.services.dynamodbv2.model.AttributeValue;
+import com.amazonaws.services.dynamodbv2.model.ScanRequest;
+import com.amazonaws.services.dynamodbv2.model.ScanResult;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class QppDataRestorationApplication {
+	private static final Logger RESTORATION_LOG = LoggerFactory.getLogger(QppDataRestorationApplication.class);
+
+	private static final AmazonDynamoDB DYNAMO_CLIENT =
+		AmazonDynamoDBClientBuilder.standard().withRegion(Regions.US_EAST_1).build();
+
+	public static AmazonDynamoDB getDynamoClient() {
+		return DYNAMO_CLIENT;
+	}
+
+	public static void main(String[] args) {
+		if (args.length == 2) {
+			List<Map<String, AttributeValue>> exportTableData = exportData(args[0]);
+			importData(exportTableData, args[1]);
```

Comment:
We aren't going to run into any memory issues exporting all the data to an in-memory data structure are we?

Reply:
How much data are we looking at?!

Reply:
If I recall correctly, @halprin said we have about 15,000 or so items. I'll have to ask him to double check but I believe that would be around 20MB of data. I do not think it is of much concern. My 3,000 items test was about 4 MB of data. If necessary, I can change the script to import while exporting. Then, we'd only handle 100 item sized list at a time.

Reply:
No worries then.

Reply:
The entire table is 37.85 MB.

Reply:
hah, definitely no worries.